transmart-api-server/README.md

```diff
@@ -0,0 +1,95 @@
+# TranSMART API server
```

Comment:
Part of this README could be also added or moved the [transmart-core docs](https://github.com/thehyve/transmart-core/tree/dev/docs)